dragonite/config.toml.example

```diff
@@ -38,12 +40,17 @@ bearer_token = "KOJI_SECRET"
 [tuning]
 #recycle_gmo_limit = 4900
 #recycle_encounter_limit = 9900
-#recycle_invasion_limit = 940
-#recycle_on_jail=false
-#minimum_account_reuse_hours = 169 # default 0
-#location_delay = 0
-#fort_location_delay = 0
-#scout_age_limit = 30
+#recycle_invasion_limit = 940  # Only works in testing builds at the moment.
+#minimum_account_reuse_hours = 169 # Default is 0
+#scout_age_limit = 30 # Default is 30 minutes.
+recycle_on_jail=false
+location_delay = 3000 # Default is 0 milliseconds.
+fort_location_delay = 3000 # Default is 0 milliseconds.
+# Automatically test accounts for being shadow banned and marked as banned in database.
+test_shadow_banned=true
```

Comment:
this will be changed to `true` by default in the next/next-next release, maybe worth just mentioning in the docs?